doc/project/dev_env_and_tools.rst

```diff
@@ -347,27 +350,34 @@ The PR is a backport or should be backported.
 
 The PR has licensing issues which require a licensing expert to review it.
 
+Depending on the PR complexity, an indication of how long a merge should be held
+to ensure proper review can found at the section on :ref:`review process <review_time>`.
+
 Labels applicable to both pull requests and issues
 ==================================================
 
-* ``Area: *``
+* ``Area:*``
 
-Indicates subsystems (e.g., Kernel, I2C, Memory Management), project functions
-(e.g., Debugging, Documentation, Process), or other categories
-(e.g., Coding Style, MISRA-C) affected by the bug or pull request.
+Indicates subsystems (e.g., "Area: Kernel", "Area: I2C", "Area: Memory
+Management"), project functions (e.g., "Area: Debugging", "Area: Documentation",
+"Area: Process"), or other categories (e.g., "Area: Coding Style", "Area:
+MISRA-C") affected by the bug or pull request.
 
 An area maintainer should be able to filter by an area label and find all issues
 and PRs which relate to that area.
 
-* ``Platform: *``
+* ``Platform:*``
 
 An issue or PR which affects only a particular platform.
 
 * ``dev-review``
+
+The issue or PR is to be discussed in a dev-review meeting.
+
 * ``TSC``
 
-The issue is to be discussed in the following `dev-review/TSC meeting`_ if time
-permits.
+The issue or PR is to be discussed in the following `dev-review/TSC meeting`_ if
+time permits.
```

Comment:
Did you mean to re-word this to just say TSC?

Reply:
Yes, you guessed right, that was my intention. It is now fixed in https://github.com/zephyrproject-rtos/zephyr/pull/44774/commits/0fd42484f58a06739a1b4c731bf80c76e8138d8d .